src/Lean/Parser/Basic.lean

```diff
@@ -549,6 +549,56 @@ partial def finishCommentBlock (nesting : Nat) : ParserFn := fun c s =>
 where
   eoi s := s.mkUnexpectedError (pushMissing := pushMissingOnError) "unterminated comment"
 
+def checkColZero (errorMsg : String) : ParserFn := fun c s =>
+    let pos      := c.fileMap.toPosition s.pos
+    if pos.column = 0 then s
+    else s.mkError errorMsg
+
+def parseStartDiff (errorMsg : String) : ParserFn :=
+  let p := satisfyFn (· == '<') errorMsg
+  andthenFn (checkColZero errorMsg)
+    (andthenFn p (andthenFn p (andthenFn p (andthenFn p (andthenFn p (andthenFn p (andthenFn p
```

Comment:
What do you think of starting the diff with 6, rather than 7, `<`?

This means that you can remove the first, exclude all editor diff-related noise, and view a "clean" file?